### PR TITLE
Moving srand to MooseApp

### PR DESCRIPTION
**Pull request by aeslaughter**
(refs #5488)

### CODE REVIEW

**Comment by friedmud** [comment]
Let's just remove `srand()` all together.

File: framework/src/base/MooseApp.C [hunk 1]
```diff
@@ -170,6 +170,7 @@ MooseApp::MooseApp(const std::string & /*deprecated_name*/, InputParameters para
     _check_input(getParam<bool>("check_input")),
     _restartable_data(libMesh::n_threads())
 {
+  srand(processor_id());
```
Comment (friedmud) [inline framework/src/base/MooseApp.C]: If we're removing it - why is it here?

Comment (permcody) [inline framework/src/base/MooseApp.C]: Are you having a conversation with yourself again? Lol

Comment (friedmud) [inline framework/src/base/MooseApp.C]: No   ?

@aeslaughter 's commit message says "Remove srand"... but then it's still in his commit.

**Comment by permcody** [comment]
@aeslaughter - go ahead and take the `srand` call out of stork.

**Comment by aeslaughter** [comment]
It will be removed when we change stork for the constructors next week.

**Pull request merged**